.github/workflows/python-ci.yml

```diff
@@ -16,19 +16,39 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+
+    - name: Install pipenv
```

Comment:
We are installing `cdo` here as well, it may be worthwhile to generalize the section name.